tests/unit/test_utils.py

```diff
@@ -99,3 +99,49 @@ def test_flatten_list(value: List, anticpated_result: List) -> None:
     """test for flatten list"""
     actual_result = utils.flatten_list(value)
     assert list(actual_result) == anticpated_result
+
+
+class HumanTimeTestData(NamedTuple):
+    """data for human time test"""
```

Comment:
```suggestion
    """Data for human time test."""
```

Reply:
Thanks, will be good to get a linter check for this in the future!